spec/multisite/distributed_cache_spec.rb

```diff
@@ -8,7 +8,7 @@ def cache(name, namespace: true)
   context "without namespace" do
     let(:cache1) { cache("test", namespace: false) }
 
-    it "does not leak state across multisite" do
+    it "does leak state across multisite" do
```

Comment:
I would describe this as using a global key in Redis or something similar rather than saying it leak state xD